packages/atomic/cypress/e2e/search-box/search-box-selectors.ts

```diff
@@ -8,6 +8,8 @@ export const SearchBoxSelectors = {
   submitButton: () =>
     SearchBoxSelectors.shadow().find('[part="submit-button"]'),
   querySuggestions: () => SearchBoxSelectors.shadow().find('[data-query]'),
+  querySuggestion: (query: string) =>
+    SearchBoxSelectors.shadow().find(`[data-query="${query}"]`),
```

Comment:
Another test failed, this time it complained that the suggestion at index 1 was not "Recent query 1", but instead "Recent query 0".

I'm guessing this is just something weird going on with the order in which elements are appended in the DOM and I'm eliminating the index from the test.

This is, as far as I'm aware, the only test dependent on the order of suggestions, so if you prefer, I can undo those changes.